src/contexts/Filter/FilterCompose.jsx

```diff
@@ -0,0 +1,71 @@
+export const filterbyPriority = ({ filterByPriority }, array) => {
```

Comment:
Liked the way you have consistenly framed out your functions using Switch Case instead of traditional if else formats!!